packages/eas-cli/src/commands/workflow/list.ts

```diff
@@ -0,0 +1,68 @@
+import EasCommand from '../../commandUtils/EasCommand';
+import { EasJsonOnlyFlag } from '../../commandUtils/flags';
+import { AppQuery } from '../../graphql/queries/AppQuery';
+import Log from '../../log';
+import formatFields from '../../utils/formatFields';
+
+export type WorkflowResult = {
+  id: string;
+  name?: string | null | undefined;
+  fileName: string | null;
+  createdAt: string | null;
+  updatedAt: string | null;
+};
+
+export default class ProjectWorkflowList extends EasCommand {
+  static override description = 'List workflows for the current project';
+
+  static override flags = {
+    ...EasJsonOnlyFlag,
+  };
+
+  static override contextDefinition = {
+    ...this.ContextOptions.ProjectId,
+    ...this.ContextOptions.LoggedIn,
+  };
+
+  async runAsync(): Promise<void> {
+    const { flags } = await this.parse(ProjectWorkflowList);
+    const {
+      projectId,
+      loggedIn: { graphqlClient },
+    } = await this.getContextAsync(ProjectWorkflowList, {
+      nonInteractive: true,
+    });
+
+    const byId = await AppQuery.byIdWorkflowsAsync(graphqlClient, projectId);
+    if (!byId) {
+      throw new Error(`Could not find project with id: ${projectId}`);
+    }
```

Comment:
(after other suggested change)
```suggestion
    const workflows = await AppQuery.byIdWorkflowsAsync(graphqlClient, projectId);
```

(`byId`/`workflows` is never falsy, can remove error)